just_audio_background/lib/just_audio_background.dart

```diff
@@ -358,17 +351,17 @@ class _PlayerAudioHandler extends BaseAudioHandler
   List<MediaItem>? get currentQueue => queue.nvalue;
 
   _PlayerAudioHandler(String playerId) {
-    _init(playerId);
+    unawaited(_init(playerId));
   }
 
   Future<void> _init(String playerId) async {
     final player = await _platform.init(InitRequest(id: playerId));
     _playerCompleter.complete(player);
-    final playbackEventMessageStream = player.playbackEventMessageStream;
-    playbackEventMessageStream.listen((event) {
-      _justAudioEvent = event;
-      _broadcastState();
-    });
+    final playbackEventMessageStream = player.playbackEventMessageStream
+      ..listen((event) {
```

Comment:
Is this for a lint, or is it your personal preference?

Reply:
This is `cascade_invocations` lint.

Reply:
If that lint is turned on (which I didn't realise it was), then let's turn it off. Blindly using cascading invocations does not add to clarity.